pytorch_lightning/plugins/training_type/tpu_spawn.py

```diff
@@ -206,7 +206,7 @@ def post_dispatch(self) -> None:
         # restore main state with best weights
         best_path = self.mp_queue.get()
         last_path = self.mp_queue.get()
-        results = self.mp_queue.get()
+        self._results = self.mp_queue.get()
```

Comment:
does it mean that we have not used it in past? @tchaton

Reply:
Yes, it was None by default I guess.

Reply:
Just need to update the test